SPECIFICATION.md

```diff
@@ -51,17 +51,18 @@ The current state of the API allows multiple authentication credentials. The cli
 
 - Must allow a combination (one or more) of the following credential types:
     - Must allow an API Key and Secret (current API)
-    - Must allow a signature secret (use for _some_ API requests, and to validate WebHook signatures)
-    - Must allow multiple private keys for JWT authentication (replacing key and secret)
-    - May allow OAuth1 token for authentication
-- Must not allow more than one credential per type.
+    - Should allow a signature secret (use for _some_ API requests, and to validate WebHook signatures)
+    - Must allow a private key for JWT authentication (replacing key and secret)
```

Comment:
Perhaps, how about:
> Must allow a private key for JWT authentication (eventually replacing key and secret)

Point was that JWT should be more and more used, as it replaces other auths. Relates to @timcraft's note below.